.travis.yml

```diff
@@ -1,14 +1,15 @@
 language: php
-sudo: false
+
 php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
+  - 7.3
+  - 7.4snapshot
```

Comment:
Any chance to not have to update this every time a new minor PHP release comes by?

I'd prefer if it just grabbed the new PHP releases as they arrive; even if that was a snapshot release one.

Reply:
There seems not to be a way to do this on travis. 7.4snapshot could be replaced by "nightly", but that should be added to the allowed-to-fail tests then, because it might actually be broken sometimes.

Reply:
As I remember "nightly" is already PHP 8.0 on Travis.

Reply:
Nightly with allowd-to-fail would be fine by me.